script.js

```diff
@@ -36,7 +36,7 @@ colorApp.getUserChoice = function() {
     const moodButton = document.querySelectorAll('.moods');
     moodButton.forEach( (button) => {
         button.addEventListener('click', function (event) {
-            const hueChoice = `/api/random/${event.target.value}`;
+            const hueChoice = `/scheme`;
```

Comment:
```suggestion
```
Connected to my previous suggestion: with scheme added in the endpoint, this line is no longer necessary